perfmetrics/scripts/fio_logger.py

```diff
@@ -0,0 +1,25 @@
+"""This script takes the logging directory path as input and 
```

Comment:
comments formatting is not proper. please fix it.

Reply:
Done